source/guides/maintenance/custom-app-container.rst

```diff
@@ -0,0 +1,402 @@
+.. _custom-app-container:
+
+Create customized application container images
+##############################################
+
+This guide describes how to edit |CL-ATTR| container base images to customize
+an application container image.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Overview
+********
+
+The official |CL-ATTR| container base images are published on Docker\* Hub and

```

Comment:
They are 2 different things so separate thoughts. The "official" CL docker image (i.e. https://hub.docker.com/_/clearlinux) is different from these CL-based application container images (i.e. https://hub.docker.com/u/clearlinux).

Reply:
Agreed. They're separate. What I struggle with as a reader in this paragraph is: Why do I need to know this information (in original)? If you want me, as a reader, to explore these links because they're relevant to the guide, then tell me why. Both links can be included. For example, you could state: "In this guide, we use the "official" CL Docker image for the first examples. Then we use an already `published image on Docker Hub [url]` for the last example.

I'd like to see the Overview tied to the examples in the guide.